Xamarin.Forms.Platform.UAP/Shell/ShellItemRenderer.cs

```diff
@@ -370,6 +370,10 @@ void UpdateNavBarVisibility()
 		void UpdatePageTitle()
 		{
 			_Title.Text = DisplayedPage?.Title ?? ShellSection?.Title ?? "";
+			if(ShellItem.Navigation.NavigationStack.Count > 1)
```

Comment:
_Title.Padding = new UwpThickness(ShellItem.Navigation.NavigationStack.Count > 1 ? 45 : 0, 0, 0, 0);
or 
_Title.Padding = ShellItem.Navigation.NavigationStack.Count > 1 ? new UwpThickness( 45, 0, 0, 0) : new UwpThickness(0, 0, 0, 0);

Reply:
What about:

```csharp
var leftPad = ShellItem.Navigation.NavigationStack.Count > 1 ? 45 : 0;
_Title.Padding = new UwpThickness(leftPad, 0, 0, 0);
```

And that 45? Is that a magic number? Will it ever change? Ever?

Reply:
It was scientifically created by changing the value until it lined up :-)

I didn't worry about it too much because these values (I find) are very often hardcoded into UWP

For example that back button has a hardcoded size of 40
https://github.com/microsoft/microsoft-ui-xaml/blob/8f1a6824d3bda36b2fdaaf25ca848d62f5c19a38/dev/NavigationView/NavigationView.cpp#L80

I still haven't quite tracked down why this is needed in the first place :-/ but I don't want that detail to block this PR

I'm not sure why when there's a back button the title grid doesn't have left padding whereas when it's a hamburger it does. Without this padding the title overlaps the back button when it's present

Reply:
Oh there it is...

Here's the better code for handling this
https://github.com/xamarin/Xamarin.Forms/pull/11410/commits/e968d041433d28fb05b4b037bd412b6970af1508

Still using the "magic" numbers that @dotMorten originally setup.

Reply:
![giphy-downsized-large](https://user-images.githubusercontent.com/1378165/87713927-d524ca00-c75f-11ea-8d14-075a0887064b.gif)